### PR TITLE
feat: update gnbSearchDropdown architecture

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### CODE REVIEW

File: apps/web/src/common/modules/navigations/gnb/modules/gnb-search-clone/modules/gnb-search-dropdown/modules/GNBSearchEmpty.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+});
+</script>
+
+<template>
+    <p-empty
+        v-if="props.inputText"
+        show-image
+    >
+        <template #image>
+            <img src="../../../../../../../../assets/images/illust_ghost.svg"
```
Comment (skdud4659) [inline apps/web/src/common/modules/navigations/gnb/modules/gnb-search-clone/modules/gnb-search-dropdown/modules/GNBSearchEmpty.vue:21]: :(

**Review by skdud4659 — APPROVED** [review]
LGTM~!

File: apps/web/src/common/modules/navigations/gnb/modules/gnb-search-clone/modules/gnb-search-dropdown/modules/GNBSearchEmpty.vue (hunk 1 above)
Comment (piggggggggy) [reply apps/web/src/common/modules/navigations/gnb/modules/gnb-search-clone/modules/gnb-search-dropdown/modules/GNBSearchEmpty.vue:21]: holy...

File: apps/web/src/common/modules/navigations/gnb/modules/gnb-search-clone/GNBSearch.vue [hunk 2]
```diff
@@ -420,13 +418,15 @@ watch(() => state.visible, async (visible) => {
         </g-n-b-search-dropdown>
         <div v-if="state.visible"
              class="background-block"
+             @click="hideSearchMenu"
```
Comment (piggggggggy) [inline apps/web/src/common/modules/navigations/gnb/modules/gnb-search-clone/GNBSearch.vue:421]: Suggestion: I think our convention of handler name should start with `handle~` prefix

File: apps/web/src/common/modules/navigations/gnb/modules/gnb-search-clone/modules/gnb-search-dropdown/modules/GNBSearchEmpty.vue [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            <br>{{ $t('COMMON.GNB.SEARCH.NO_RESULT_2') }}
+        </p>
+    </p-empty>
+    <p-empty
+        v-else-if="props.isRecent"
+        show-image
+        image-size="md"
+    >
+        <template #image>
+            <img src="../../../../../../../../assets/images/illust_microscope.svg"
```
Comment (piggggggggy) [inline apps/web/src/common/modules/navigations/gnb/modules/gnb-search-clone/modules/gnb-search-dropdown/modules/GNBSearchEmpty.vue:40]: 🤔

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Pull request merged**